ecpy_hqc_legacy/tasks/tasks/instr/spdev_tasks.py

```diff
@@ -29,6 +27,10 @@ class DemodSPTask(InstrumentTask):
     """Get the averaged quadratures of the signal.
 
     """
+
+    # return averaged data or single shot data
+    average = Unicode('Avg before demod').tag(pref=True)
```

Comment:
Please use an Enum instead to enforce possible values and document their meaning.